kratos/python_interface/kratos_globals.py

```diff
@@ -81,4 +81,14 @@ def HasVariable(self,VarName):
         else:
             return False
 
+    def GetConstitutiveLaw(self, ConstitutiveLawName):
```

Comment:
question: wouldn't it be better to directly call the kernel function?

Reply:
How would you call it from outside then?
I think the call would not be through KratosGlobals any more?

Reply:
we could call the kernel directly. However my comment is really of secundary importance. just dismiss it for now.